_authors/ashisagr.markdown

```diff
@@ -0,0 +1,7 @@
+---
+name: Ashish Agrawal
+short_name: ashiasgr
+photo: '/assets/media/authors/ashisagr.jpeg'
```

Comment:
The uploaded file ends in `.jpg` - this will have to match. One of the two will have to change. :-)

Reply:
@nateynateynate I fixed it. Thanks for catching it.